### PR TITLE
[Console] Deprecate `$defaultName` and `$defaultDescription`

### PR DESCRIPTION
**Pull request by alexandre-daubois**

### CODE REVIEW

File: console.rst [hunk 1]
```diff
@@ -171,6 +171,12 @@ You can optionally define a description, help message and the
     classes, but it won't show any description for commands that use the
     ``setDescription()`` method instead of the static property.
 
+.. deprecated:: 6.1
+
+    The static property ``$defaultDescription`` was deprecated in Symfony 6.1.
+    Declare your command description with the ``#[AsCommand]`` attribute
```
Comment (94noni) [inline console.rst:177]:
```suggestion
    Declare your optional command description with the ``#[AsCommand]`` attribute
```

**Review by 94noni — APPROVED** [review]

File: console.rst (hunk 1 above)
Comment (OskarStark) [reply console.rst:177]: And please end this sentence with a dot 👍

Comment (javiereguiluz) [reply console.rst:177]: Updated while merging. Thanks.

**Comment by javiereguiluz** [comment]
Thanks Alex!

**Pull request merged**